### PR TITLE
LV modular maps

### PR DESCRIPTION
**Pull request by heffy117**
adds a few modular maps to LV

## About The Pull Request
Adds two new modular map zones to the map along with 4 modular maps to go with them.

## Why It's Good For The Game
More variation.

## Changelog
:cl:
fix: fixed a few things
/:cl:

### CODE REVIEW

**Comment by chesse20** [comment]
can you put some screenshots in a image gallery so can quickly look at these modular maps

**Comment by TiviPlus** [comment]
Use mapdiffbot lazy ass

File: code/modules/mapping/modular_mapping.dm [hunk 1]
```diff
@@ -162,6 +162,54 @@
 	template_height = 15
 	keepcentered = TRUE
 
+/datum/map_template/modular/lv624/lvhydrobridge1
+	name = "LV Hydro Bridge One"
```
Comment (TiviPlus) [inline code/modules/mapping/modular_mapping.dm:166]: Dont use numbers in typepaths please

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Rest ok, ripgrayson pr

**Comment by heffy117** [comment]
> Use mapdiffbot lazy ass

What and how. Am new.

**Comment by TiviPlus** [comment]
> > Use mapdiffbot lazy ass
> 
> What and how. Am new.

Not you Chesse we literally have a bot for this

**Review by RipGrayson — CHANGES_REQUESTED** [review]
To start with, unless you're doing a circular inserts, map spawners belong in corners, like so.
court10
court11

There's two problems here, one's a nitpick and the other is more serious.
I don't like the huge open sand area here, I suggest you expand the river a couple tiles to the west.
The bigger problem is that the AI nodes here don't link up. We can't path through solid objects, and some of the nodes are either embedded in walls or otherwise obstructed by solid walls.
court14

This one isn't bad, but because the area circled is closed off the AI is unable to actually enter the temple from the south, please fix that.
court15

Two things here, the circled area looks kinda jank, please replace the corners with the robots with proper stair corners.
The building is also rather small, it needs an exit on the other side or some poor queen is going to get trapped in it while retreating and salt up #mapping.
court13

**Review by RipGrayson — APPROVED** [review]
This looks fine to me, good job.

**Pull request merged**